arch/c4_context.md

```diff
@@ -6,40 +6,70 @@ All production Systems (External and Radix) and Users that are commonly found on
 
 ![c4_context.puml](http://www.plantuml.com/plantuml/proxy?cache=no&fmt=svg&src=https://raw.githubusercontent.com/radixdlt/docs/arch/arch/c4_context.puml)
 
+# Universes
+
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/).
+
+From an architectural point of view, a Universe identifies and overlay network on top of Internet or in a LAN.
+Core/Boot nodes can be configured to be part of a specific universe.
+
+# Atoms and Particles
+
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/).
+
+On an Architectural level an Atom is totally ordered sequence of Transactions (Particles) that must be validated as a unit and committed to the Ledger (DB) atomically. 
+
+The minimal unit of transactional data transferred over wire (sync/gossip/WebSocket) is thus an Atom.
+
 # Shards and Chunks
 
-Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/), shards enable Radix Networks to scale infinitely.
-This is achieved by splitting up global state (Ledger) to different nodes. Each node can be part of one or more Shards slices - called Chunks.
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/).
+
+Sharding enable Radix Networks to have unlimited scalability.
+This is achieved by splitting up global state (Ledger) to different nodes in a deterministic way.
+The goal with this sharding scheme is to be able to tell where certain Particle (transaction) goes just by looking at it, so that we can check double spends (and other attack vectors) efficiently.
 
 The minimal Shard space that a nodes can serve is one Chunk. The number of Chunks served depends on the hardware configuration (CPU, MEM, I/O) of
 a specific node. Each node has advanced algorithms for monitoring its own performance and dynamically - to other nodes non-deterministically - scale up and down
  the number of chunks served.
 
-## Redundancy
+Nodes in an overlapping Shard space are redundant to each other, which means that they persist and validate the same transactions.
 
-Nodes within the same Shard space are redundant to each other, which means that they serve and validate the same transactions.
+# Mass Consensus
 
-# Core Nodes
+[Whitepaper v2](https://papers.radixdlt.com/tempo/v2/)
 
-The first class citizens of the Radix Network. Nodes are organised in Shards. A node can serve ("be in") one or more Shard Chunks.
+Mass is a Node property. Well-behaved Nodes of a Radix Network gain mass over time. Mass determines the power each node has in a Consensus agreement at a given time.
```

Comment:
in a consensus agreement where temporal resolution is not possible - I think that's important to highlight, usually (without netsplits) we don't need mass at all

Reply:
I realise now that this is specifically under the "Mass Consensus" section, so my point is not quite as important. But still worth mentioning I think, just to be perfectly clear.